From stable-bounces@linux.kernel.org  Sat Jun 18 13:22:38 2005
Date: Sat, 18 Jun 2005 23:22:28 +0300
From: Mika Kukkonen <mikukkon@gmail.com>
To: hch@infradead.org
Cc: torvalds@osdl.org, linux-kernel@vger.kernel.org
Subject: [PATCH 2.6.12] Fix typo in drivers/pci/pci-driver.c

The git commit 794f5bfa77955c4455f6d72d8b0e2bee25f1ff0c
accidentally suffers from a previous typo in that file
(',' instead of ';' in end of line). Patch included.

Signed-off-by: Mika Kukkonen <mikukkon@iki.fi>
Signed-off-by: Chris Wright <chrisw@osdl.org>

Index: linux-2.6/drivers/pci/pci-driver.c
===================================================================
--- linux-2.6.orig/drivers/pci/pci-driver.c	2005-06-18 22:05:42.642463416 +0300
+++ linux-2.6/drivers/pci/pci-driver.c	2005-06-18 22:10:37.486761280 +0300
@@ -396,7 +396,7 @@
 	/* FIXME, once all of the existing PCI drivers have been fixed to set
 	 * the pci shutdown function, this test can go away. */
 	if (!drv->driver.shutdown)
-		drv->driver.shutdown = pci_device_shutdown,
+		drv->driver.shutdown = pci_device_shutdown;
 	drv->driver.owner = drv->owner;
 	drv->driver.kobj.ktype = &pci_driver_kobj_type;
 	pci_init_dynids(&drv->dynids);
